.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+name: CI
+
+jobs:
+  test:
+    name: run tests
+    runs-on: ubuntu-latest
```

Comment:
I usually configure a specific Ubuntu version for more reproducibility.

Reply:
Updated!